Paper.tex

```diff
@@ -1197,7 +1198,7 @@ \subsection{Mining Proof-of-Work} \label{ch:pow}
 
 One plague of the Bitcoin world is ASICs. These are specialised pieces of compute hardware that exist only to do a single task. In Bitcoin's case the task is the SHA256 hash function. While ASICs exist for a proof-of-work function, both goals are placed in jeopardy. Because of this, a proof-of-work function that is ASIC-resistant (i.e. difficult or economically inefficient to implement in specialised compute hardware) has been identified as the proverbial silver bullet.
 
-Two directions exist for ASIC resistance; firstly make it sequential memory-hard, i.e. engineer the function such that the determination of the nonce requires a lot of memory and bandwidth such that the memory cannot be used in parallel to discover multiple nonces simultaneously. The second is to make the type of computation it would need to do general-purpose; the meaning of ``specialised hardware''  for a general-purpose task set is, naturally, general purpose hardware and as such commodity desktop computers are likely to be pretty close to ``specialised hardware'' for the task. For Ethereum 1.0 we have chosen the first path.
+Two directions exist for ASIC resistance; firstly make it sequential memory-hard, i.e. engineer the function such that the determination of the nonce requires a lot of memory and bandwidth such that the memory cannot be used in parallel to discover multiple nonces simultaneously. The second is to make the type of computation it would need to do general-purpose; the meaning of ``specialised hardware''  for a general-purpose task set is, naturally, general purpose hardware and as such commodity desktop computers are likely to be pretty close to ``specialised hardware'' for the task. For Ethereum 1.0 we have chosen the first path.\footnote{Ethereum 1.0 uses proof-of-work, while it is planned that Ethereum 2.0 will include \href{https://github.com/ethereum/wiki/wiki/Proof-of-Stake-FAQ}{proof-of-stake} and \href{https://github.com/ethereum/wiki/wiki/Sharding-FAQ}{sharding}.}
```

Comment:
Future protocol updates are not part of the specification.

Reply:
OK, but if you are going to say "Ethereum 1.0", it needs to be defined, while it isn't now. This PR defines it while also putting Ethereum 1.0 in the context of planned future protocol updates. While it may not be a formal spec, it is briefly in a footnote, with links for further info.

Reply:
The discussion about the definition is opened here https://github.com/ethereum/yellowpaper/issues/217.